pandas/tseries/tests/test_timeseries.py

```diff
@@ -4441,6 +4441,15 @@ def test_nat_operations(self):
         self.assertEqual(s.min(), exp)
         self.assertEqual(s.max(), exp)
 
+    def test_round_nat(self):
```

Comment:
Does exist a way to check a message of exception? Or should I just check for `AttributeError`?